cmd/node/main.go

```diff
@@ -197,12 +199,30 @@ func main() {
 	logger.Info("Got SIGTERM signal")
 }
 
+func obtainNodeIDWithRetries(client k8sClient.Client, featureConf featureconfig.FeatureChecker,
+	logger *logrus.Logger) (string, error) {
+	var (
```

Comment:
Just use variables in `(nodeID string, err error)`, no need variable declaration

Reply:
Do you mean in `if` statement?

Reply:
No, I mean you can use [named resutlt parameters](https://golang.org/doc/effective_go#named-results):
```golang
func obtainNodeIDWithRetries(client k8sClient.Client, featureConf featureconfig.FeatureChecker,
   logger *logrus.Logger) (nodeID string, err error) {
	for i := 0; i < numberOfRetries; i++ {
        // other code goes here
```

Reply:
Ok, thanks! Refactored.